mongodb-odbc-driver/bin/platforms.sh

```diff
@@ -55,12 +73,14 @@ win64)
     EXTRACT='unzip'
     PLATFORM_ARCH='64'
     PLATFORM_NAME='windows'
-    CMAKE_GENERATOR="Visual Studio 14 2015 Win64"
+    #CMAKE_GENERATOR="Visual Studio 14 2015 Win64"
+    CMAKE_GENERATOR="Visual Studio 16 2019"
     # make sure binaries we use in our scripts are available in the PATH
-    DEVENV_PATH='/cygdrive/c/Program Files (x86)/Microsoft Visual Studio 14.0/Common7/IDE'
+    #DEVENV_PATH='/cygdrive/c/Program Files (x86)/Microsoft Visual Studio 14.0/Common7/IDE'
+    DEVENV_PATH='/cygdrive/c/Program Files (x86)/Microsoft Visual Studio/2019/Professional/Common7/IDE'
```

Comment:
This is fine as it is, if you want to get the path programmatically that may avoid needing to update this in the future.  Something like this should work. 
```suggestion
    DEVENV_PATH="$(vswhere.exe -latest -prerelease -products Microsoft.VisualStudio.Product.Professional -property productPath | cygpath -u -f - | sed 's/\/[^/]*$//')"
```